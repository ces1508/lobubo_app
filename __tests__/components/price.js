import React from 'react'
import Price from '../../src/components/price'
import rendered from 'react-test-renderer'

test('render correctly', () => {
  const tree = rendered.create(<Price value={25000} />)
  expect(tree).toMatchSnapshot()
})