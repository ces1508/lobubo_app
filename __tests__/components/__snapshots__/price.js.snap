// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render correctly 1`] = `
<Text>
  $25,000
   
</Text>
`;
